src/Administration/Resources/app/administration/src/app/mixin/listing.mixin.ts

```diff
@@ -85,7 +87,11 @@ export default Shopware.Mixin.register(
                     return {};
                 }
 
-                return this.searchRankingService.getSearchFieldsByEntity(this.searchConfigEntity);
+                if (!this._cachedSearchRankingFields) {
```

Comment:
this only cache the request when loading the listing, try to search in the search bar, it still fetch the search-preference which is not ideal

I'd cache the request in the cache-adapter if possible, or at least in the `searchRankingService.getSearchFieldsByEntity`